index.php

```diff
@@ -10,10 +10,17 @@
     ],
     'hooks' => [
         'page.render:after' => function ($contentType, $data, $html) {
-            if ($contentType === 'html') {
-                $html = str_replace('</body>', snippet('admin-bar', [], true), $html) . '</body>';
+            if ($contentType !== 'html') return;
```

Comment:
As far as I know, if you don't return a string then Kirby keeps the original HTML.
This lets us use early returns and avoid having to `return $html`.

If that's less clear than the original code, I can revert it.